cedana-helm/templates/secrets.yaml

```diff
@@ -10,3 +10,5 @@ metadata:
 type: Opaque
 data:
   signoz-api-key: "{{ .Values.cedanaConfig.signozApiKey | b64enc }}"
+  cedana-api-key: "{{ .Values.cedanaConfig.cedanaApiKey | b64enc }}"
+  cedana-api-server: "{{ .Values.cedanaConfig.cedanaApiServer | b64enc }}"
```

Comment:
Need to add `cedanaApiKey` or `cedanaApiServer` to values.yaml.

Reply:
done